metadata-service/configuration/src/main/resources/application.yml

```diff
@@ -275,7 +275,7 @@ bootstrap:
   upgradeDefaultBrowsePaths:
     enabled: ${UPGRADE_DEFAULT_BROWSE_PATHS_ENABLED:false} # enable to run the upgrade to migrate legacy default browse paths to new ones
   backfillBrowsePathsV2:
-    enabled: ${BACKFILL_BROWSE_PATHS_V2:false} # Enables running the backfill of browsePathsV2 upgrade step. There are concerns about the load of this step so hiding it behind a flag. Deprecating in favor of running through SystemUpdate
+    enabled: ${BACKFILL_BROWSE_PATHS_V2:true} # Enables running the backfill of browsePathsV2 upgrade step. There are concerns about the load of this step so hiding it behind a flag. Deprecating in favor of running through SystemUpdate
```

Comment:
do we want to enable this by default? I think this should be an opt in thing for users only if they're confident this backfill job can handle the load

Reply:
Will do!